application/server/server.go

```diff
@@ -477,28 +477,42 @@ func textDocumentDidOpenHandler() jrpc2.Handler {
 	})
 }
 
+func isDotSnykFile(uri sglsp.DocumentURI) bool {
+	return strings.HasSuffix(string(uri), ".snyk")
+}
+
+func dotSnykFileDidChangeScan(ctx context.Context, workspaceFolderPath *workspace.Folder) {
+	go workspaceFolderPath.ScanFolder(ctx)
```

Comment:
This is a potential panic, if the pointer is nil (e.g. when the file is not contained in the workspaceFolder.